AutomatedTesting/Gem/PythonTests/Prefab/TestSuite_Periodic.py

```diff
@@ -18,5 +18,15 @@ class TestAutomationNoAutoTestMode(EditorTestSuite):
     # Enable only -BatchMode for these tests. Some tests cannot run in -autotest_mode due to UI interactions
     global_extra_cmdline_args = ["-BatchMode"]
 
-    class test_DeletePrefab_DuplicatedPrefabInstance(EditorSharedTest):
-        from .tests.delete_prefab import DeletePrefab_DuplicatedPrefabInstance as test_module
\ No newline at end of file
+    class test_AddEntity_UnderLevelPrefab(EditorSharedTest):
+        from .tests.add_entity import AddEntity_UnderLevelPrefab as test_module
+
+    class test_AddEntity_UnderAnotherEntity(EditorSharedTest):
+       from .tests.add_entity import AddEntity_UnderAnotherEntity as test_module
+
+    class test_AddEntity_UnderAPrefab(EditorSharedTest):
+        from .tests.add_entity import AddEntity_UnderAPrefab as test_module
```

Comment:
minor suggest: test_AddEntity_UnderPrefab

Reply:
Re-added missing test data and updated test name with feedback.

Reply:
Re-added missing test data and updated test name with feedback.